app/views/service_mailer/notify_owner_email.text.erb

```diff
@@ -1,11 +1,11 @@
 # Your changes to <%= @service.name %> have been approved.
 
 
-<% if @service.publicly_visible? %>
+<% if @service.publicly_visible? && ENV['SCOUT_URL'] %>
 
-They're now publicly visible in the Family Information Service directory:
+They're now publicly visible in the directory:
```

Comment:
suuuper persnickety but I think this would sound better as `They are` 😅

Reply:
Your wish is my command! 🪄 😄